### ISSUE TITLE
Mapserver wrong to position label in legend when maxlength and wrap

### ISSUE THREAD

**Issue by aperi2007** [MapServer/MapServer#4620]
Hi,
I notice that mapserver 6.3-dev
wrong to position the label in a legend image when it is set the two parameters maxlenght and wrap.

As example:
This
  LEGEND
    IMAGECOLOR 255 255 255
    STATUS OFF
    KEYSIZE 15 20
    KEYSPACING 3 3
    POSTLABELCACHE true
    # label object
    LABEL
      TYPE BITMAP
      POSITION ul
      SIZE TINY
      OFFSET 0 -10
      COLOR 0 0 0
    END
  END

This legend is returned with the label correctl put on 
"ul" position.

But If I set also
maxlenght 20
and
wrap ' '

the label is positioned wrongly in the bottom.

Please notice this bug is referred only to the labels in a legend image.
Infact the labels in the map work correctly with the maxlength and wrap options.

**Comment by ruhri** [comment]
Problem exists with Mapserver 7.0.0. too, moreover it seems to me, that POSITION and OFFSET aren't supported. Are they?

**Comment by tbonfort** [comment]
@ruhri please be more precise. position and offset aren't meant to be supported in the legend rendering

**Comment by ruhri** [comment]
I thought, the above behaviour described by aperi2007 was a bug in vertical alignment of text in relation to legendsymbols, I didn't know, that it isn't supported ...
It seems, that legend-text is positioned on a symbol-"baseline". IMHO this should be configurable (bootom|center|up?), specially larger legend-symbols in combination with small text doesn't look that clean. would that enhancement be possible?

**Comment by tbonfort** [comment]
Yes it could be added relatively easily. You can contract a mapserver developer to implement the RFC and code for you, or do that yourself.
